pycc/local.py

```diff
@@ -25,6 +25,10 @@ class Local(object):
         used in PAO calculations for determining the norm-cutoff of the virtual space
     lindep_cut: double (optional)
         used in PAO calculations for determining linear dependence threshold
+    e_conv : float
+        convergence condition for MP2 correlation energy (default if 1e-7)
+    r_conv : float
+        convergence condition for MP2 residual rmsd (default if 1e-7)
```

Comment:
"default is 1e-7"?

Reply:
oops, typo. the default convergence _is_ 1e-12.